build.gradle

```diff
@@ -334,6 +327,14 @@ dependencies {
   contractTestCompile group: 'au.com.dius.pact.consumer', name: 'junit5', version: versions.pactVersion
   contractTestCompile group: 'au.com.dius.pact.consumer', name: 'java8', version: versions.pactVersion
 
+
+  contractTestCompile group: 'au.com.dius.pact.provider', name: 'junit5', version: '4.1.14'
+  contractTestCompile group: 'au.com.dius.pact.provider', name: 'spring', version: '4.1.14'
+  contractTestCompile group: 'au.com.dius.pact.provider', name: 'junit5spring', version: '4.1.14'
+
+  testImplementation "au.com.dius.pact.provider:junit5:4.1.7"
+  testImplementation "au.com.dius.pact.provider:junit5spring:4.1.7"
```

Comment:
we seem to be using different versions here in any case:
we should use the variable defined as versions.pactVersion which points to 4.1.7